leaf-ui/rappid-extensions/BackboneModels.js

```diff
@@ -1 +1,27 @@
-/** This file contains backbone model representations of the original model objects - WIP */
\ No newline at end of file
+/** This file contains backbone model representations of the original model objects - WIP */
+
+
+// FuncSegmentModel from FuncSegment class in modelObjects.js
+var FuncSegmentModel = Backbone.Model.extend({
+    idAttribute: "uid",
+
+    // initialize function - could also save all of the input parameters into one variable 
```

Comment:
So you actually have to save all the input parameters into one option variable I am pretty sure. You can test this by trying to create a FuncSegmentModel instance and then printing it out to see how/if the parameters were assigned properly